### ISSUE TITLE
Front End Checkbox/Radios are not keyboard accessible

### ISSUE THREAD

**Issue by kandrewsind** [wpninjas/ninja-forms#3673]
Issue in all versions

Add any number of checkbox/radio button fields to a form

When you visit the form on the front-end the checkbox/radio button fields cannot be accessed via keyboard navigation (tabbing).

For accessibility reasons users should be able to access the checkbox/radio button fields using their keyboard.

Quick example of how a stylized checkbox can be made keyboard accessible:
https://codepen.io/anon/pen/pmxMoM
